che-base/scripts/base/commands/cmd_backup.sh

```diff
@@ -12,10 +12,10 @@
 cmd_backup() {
   debug $FUNCNAME
 
-  # possibility to skip che projects backup
+  # possibility to skip ${CHE_FORMAL_PRODUCT_NAME} projects backup
   SKIP_BACKUP_CHE_DATA=${1:-"--no-skip-data"}
   if [[ "${SKIP_BACKUP_CHE_DATA}" == "--skip-data" ]]; then
-    TAR_EXTRA_EXCLUDE="--exclude=instance/data/che"
+    TAR_EXTRA_EXCLUDE="--exclude=instance/data${CHE_CONTAINER_ROOT}"
```

Comment:
AFAIK no
${CHE_CONTAINER_ROOT} is beginning with a /
it's like /che , /codenvy, etc